### PR TITLE
CSVS-64 Now the accountId is obtained using the domain vm and account name

### PR DESCRIPTION
**Pull request by MedvedevBW**

### CODE REVIEW

File: src/main/scala/com/bwsw/cloudstack/vault/server/cloudstack/CloudStackService.scala [hunk 1]
```diff
@@ -108,16 +108,24 @@ class CloudStackService(сloudStackTaskCreator: CloudStackTaskCreator,
   def getAccountIdByVmId(vmId: UUID): UUID = {
     logger.debug(s"getAccountIdByVmId(vmId: $vmId)")
 
-    val accountName = jsonSerializer.deserialize[VirtualMachinesResponse](
-      getEntityJson(vmId.toString, сloudStackTaskCreator.idParameter, Command.ListVirtualMachines)
+    val virtualMachine = jsonSerializer.deserialize[VirtualMachinesResponse](
```
Comment (placccebo) [inline src/main/scala/com/bwsw/cloudstack/vault/server/cloudstack/CloudStackService.scala]: why do you use both 'VirtualMachine' and its acronym 'VM'?

File: src/main/scala/com/bwsw/cloudstack/vault/server/cloudstack/CloudStackService.scala [hunk 2]
```diff
@@ -190,8 +197,8 @@ class CloudStackService(сloudStackTaskCreator: CloudStackTaskCreator,
     logger.debug(s"Tag was set to resource: $resourceId, $resourceType")
   }
 
-  private def getEntityJson(parameterValue: String, parameterName: String, command: Command): String = {
-    def task = сloudStackTaskCreator.createGetEntityTask(parameterValue, parameterName, command)
+  private def getEntityJson(filterRequestParameters: Map[String, String], command: Command): String = {
```
Comment (placccebo) [inline src/main/scala/com/bwsw/cloudstack/vault/server/cloudstack/CloudStackService.scala]: filterRequestParameters -> parameters

**Review by placccebo — CHANGES_REQUESTED** [review]

File: src/main/scala/com/bwsw/cloudstack/vault/server/cloudstack/CloudStackService.scala (hunk 2 above)
Comment (MedvedevBW) [reply src/main/scala/com/bwsw/cloudstack/vault/server/cloudstack/CloudStackService.scala]: ok

File: src/main/scala/com/bwsw/cloudstack/vault/server/cloudstack/CloudStackService.scala (hunk 1 above)
Comment (MedvedevBW) [reply src/main/scala/com/bwsw/cloudstack/vault/server/cloudstack/CloudStackService.scala]: fixed

**Review by placccebo — APPROVED** [review]

**Pull request merged**